enterprise/dbcrypt/dbcrypt_internal_test.go

```diff
@@ -57,7 +56,7 @@ func TestUserLinks(t *testing.T) {
 			OAuthRefreshToken: "refresh",
 			UserID:            link.UserID,
 			LoginType:         link.LoginType,
-			DebugContext:      json.RawMessage("{}"),
```

Comment:
Can we throw in a value here and check the output below just to make sure it gets marshalled and unmarshalled correctly?